backend/api/game_test.go

```diff
@@ -4,9 +4,17 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
+	"net/url"
+	// "fmt"
+	"encoding/json"
 
+	"github.com/jak103/usu-gdsf/db"
+	"github.com/jak103/usu-gdsf/models"
 	"github.com/jak103/usu-gdsf/auth"
 	"github.com/stretchr/testify/assert"
+	 "github.com/stretchr/testify/require"
+	//  "github.com/labstack/echo/v4"
 )
```

Comment:
These imports should be removed, not commented in the commit

Reply:
ohh yeah !!!

Reply:
I will take care of it